src/jsx.d.ts

```diff
@@ -1782,6 +1782,7 @@ export namespace JSXInternal {
 		crossorigin?: Signalish<HTMLAttributeCrossOrigin>;
 		crossOrigin?: Signalish<HTMLAttributeCrossOrigin>;
 		decoding?: Signalish<'async' | 'auto' | 'sync' | undefined>;
+		fetchPriority?: Signalish<'high' | 'auto' | 'low' | undefined>;
```

Comment:
Should probably also add the attribute casing while we're at it:

```suggestion
		fetchpriority?: Signalish<'high' | 'auto' | 'low' | undefined>;
		fetchPriority?: Signalish<'high' | 'auto' | 'low' | undefined>;
```